nghttp/Kconfig

```diff
@@ -0,0 +1,12 @@
+menu "NGHTTP configuration"
+
+    config ESP_NGHTTP_INBOUND_BUFFER_LENGTH
```

Comment:
Maybe we can rename it to `ESP_NGHTTP2_INBOUND_BUFFER_LENGTH`, just to be consistent with `HTTP2` naming scheme

Reply:
Fixed